app/models/meetup.server.test.ts

```diff
@@ -0,0 +1,59 @@
+// @vitest-environment node
+import { getRemixAustinInfo } from "~/models/meetup.server";
+import { server } from "../../mocks/server";
+import { rest } from "msw";
+import type {
+  MeetupEvent,
+  MeetupGroupByUrlname,
+  MeetupGroupResponse,
+  MeetupRequestBody,
+} from "~/models/meetup";
+
+const MOCK_EVENT: MeetupEvent = {
+  title: "Title",
+  shortUrl: "https://short.url",
+  venue: {
+    name: "Online",
+    address: "",
+    city: "",
+    state: "",
+  },
+  dateTime: "2022-01-01T01:00:00Z",
+  going: 100,
+};
+
+const MOCK_GROUP: MeetupGroupByUrlname = {
+  link: "https://group.link",
+  memberships: {
+    count: 200,
+  },
+  upcomingEvents: {
+    edges: [{ node: MOCK_EVENT }],
+  },
+};
+describe("meetup.server", () => {
+  describe("getRemixAustinInfo", () => {
+    let requestBody: MeetupRequestBody;
+
+    beforeEach(() => {
+      server.use(
+        rest.post("https://www.meetup.com/gql", async (req, res, context) => {
+          requestBody = await req.json();
+          const response: MeetupGroupResponse = {
+            data: { groupByUrlname: MOCK_GROUP },
+          };
+          return res(context.status(200), context.json(response));
+        })
+      );
+    });
```

Comment:
Nice! I like seeing the use of msw for the mock server.

Here might be a good place to add some tests for api failures.

Reply:
If you happen to be able to reuse [`mocks/index.js`](https://github.com/remix-austin/remixaustin-com/blob/main/mocks/index.js) _(may have to make it a `.ts`)_, would you mind adding this test to it? 🙂

Reply:
i think doing so would affect `npm run test:e2e:run`. those smoke tests expect requiring the `./mocks` to have the side effect of running `server.listen()` in a separate process. however, for these tests i'm doing the standard msw strategy of hitting that in a `beforeAll`. since i'm not too familiar with cypress i decided not to mess with it and stick to the standard msw strategy of doing this in `./mocks/server`.

Reply:
update: i didn't notice that cypress was swapped for playwright. doing so would still affect `npm run dev` in a similar fashion due to the usage of `binode --require ./mocks -- @remix-run/dev:remix dev`.

investigating more, the indie stack puts `mocks/index.js` there in order to allow for you to mock something during local dev. seems like a remix convention. (or maybe a kcd convention). so mixing the `beforeAll` stuff into the same file isn't ideal. so the test msw and this dev msw need to be different.

Reply:
> seems like a remix convention. (or maybe a kcd convention).

😅 I think lots of the Remix testing conventions are kcd things

I'm okay having them separate for now, but it does seem a bit strange to have 2 versions of the same thing.

I guess the only problem using the same msw server for both is if you were running `dev` and `test` at the same time, there could be clashes 🤷‍♂️ 

Anyway, I'm not too concerned, especially since we're not really using the mocks much (yet)